www/content/attributes/hx-on.md

```diff
@@ -6,40 +6,81 @@ The `hx-on` attribute allows you to embed scripts inline to respond to events di
 
 `hx-on` improves upon `onevent` by enabling the handling of any event for enhanced [Locality of Behaviour (LoB)](/essays/locality-of-behaviour/). This also enables you to handle any htmx event.
 
-The value is an event name, followed by a colon `:`, followed by the script:
+There are two forms of this attribute, one in which you specify the event as part of the attribute name
+after a colon (`hx-on:click`, for example), and one that uses the `hx-on` attribute directly. The
+latter form should only be used if IE11 support is required.
+
+### Forms
+#### hx-on:* (recommended)
+The event name follows a colon `:` in the attribute, and the attribute value is the script to be executed:
 
 ```html
-<div hx-on="click: alert('Clicked!')">Click</div>
+<div hx-on:click="alert('Clicked!')">Click</div>
 ```
 
-All htmx events can be captured, too!
+All htmx events can be captured, too! Make sure to use the [kebab-case event name](@/docs.md#events),
+because DOM attributes do not preserve casing.
+
+To make writing these a little easier, you can use the shorthand double-colon `hx-on::` for htmx
+events, and omit the "htmx" part:
 
 ```html
-<button hx-get="/info" hx-on="htmx:beforeRequest: alert('Making a request!')">
+<!-- These two are equivalent -->
+<button hx-get="/info" hx-on:htmx:before-request="alert('Making a request!')">
     Get Info!
 </button>
+
+<button hx-get="/info" hx-on::before-request="alert('Making a request!')">
+    Get Info!
+</button>
+
 ```
 
-### Symbols
+Adding multiple handlers is easy, you just specify additional attributes:
+```html
+<button hx-get="/info"
+        hx-on::beforeRequest="alert('Making a request!'")
```

Comment:
This should be `"alert('Making a request!')"`

It was exactly that line I copied for a test. 😄